content/organizations/keeping-your-organization-secure/reviewing-the-audit-log-for-your-organization.md

```diff
@@ -335,6 +337,23 @@ An overview of some of the most common actions that are recorded as events in th
 | `create` | Triggered when an organization member requests that an organization owner install an integration for use in the organization.
 | `close` | Triggered when a request to install an integration for use in an organization is either approved or denied by an organization owner, or canceled by the organization member who opened the request.
 
+### `ip_allow_list` category actions
+
+| Action | Description
+|------------------|-------------------
+| `enabled` | Triggered when an IP allow list was enabled for an organization.
```

Comment:
```suggestion
| `enable` | Triggered when an IP allow list was enabled for an organization.
```

@Hollywood, do you know if this should be `enable`? I haven't tested this yet, but I believe the log actions usually use `enable` rather than `enabled`.

Reply:
Hey @ethanpalm I took the event types straight from one of our [other docs](https://docs.github.com/en/github-ae@latest/admin/user-management/monitoring-activity-in-your-enterprise/audited-actions#ip-allow-lists) on the same subject.

Reply:
Thanks! It looks like `enable` is the way to go. I'll push that change and get this merged once all the checks are passing!